### DISCUSSION TITLE
Change Boot Screen Text

### DISCUSSION THREAD

**Discussion by mo-may** [bangcorrupt/hacktribe#32] [Q&A] [answered]
Is it possible & if it is, allowed? ^^

**Comment by bangcorrupt** [comment, marked as answer]
I like to tag my work, but it could be cool to have a collection of pixel art boot screens.  It's 1 bit per pixel, 128x64, inverted monochrome.  At some point in the future I'll add a script to change the boot image.

**Reply by gradientledge** [reply]
good chance to have a tip jar for a pro feature, everyones gotta eat especially since this is the only option mod wise.

**Reply by bangcorrupt** [reply]
Everything will always be free as soon as I think it works, but I can certainly add a tip button.
